cogs/hl.py

```diff
@@ -30,9 +30,20 @@ class Highlighter(AceMixin, commands.Cog):
 
 	@commands.command(aliases=['h1'])
 	@commands.bot_has_permissions(manage_messages=True, add_reactions=True)
-	async def hl(self, ctx, *, code):
+	async def hl(self, ctx, *, code: str = None):
 		'''Highlight some code.'''
-
+		if ctx.message.reference is not None:
+			# get the replied to message contents if there is one
+			ref_msg = ctx.message.reference.resolved
```

Comment:
`discord.MessageReference.resolved` was added in version 1.6, so it's not available.